tracee/tracee.go

```diff
@@ -64,15 +65,20 @@ type StringFilter struct {
 	Enabled  bool
 }
 
+type BoolFilter struct {
+	Set     bool
```

Comment:
I think "set" is confusing in this context. in other place (in flag parsing) we use it to mean "enabled". can we change it to "value" or something?

Reply:
we also might want to consider renaming enabled to "isSet" like in flag parsing but not a must